docs/Modern-GraphQLInRelay.md

```diff
@@ -220,6 +238,11 @@ This would produce three generated files, and two `__generated__` directories:
 * `src/Components/__generated__/DictionaryComponent_definition.graphql.js`
 * `src/Queries/__generated__/DictionaryQuery.graphql.js`
 
+If you use `--persist`, then an extra query map json file will also be generated:
+
+* `src/Queries/__generated__/DictionaryQuery.graphql.json`
+
+Only one query map json is generated in this instance because only concrete queries can be persisted. Fragments are not persisted. 
```

Comment:
> query map json **file**

Not sure if ‘a json’ is used as equivalent of a JSON file nowadays, just reads a little nicer to me, but I’m also a non-native reader/writer so 🤷‍♂️

Reply:
Done.